### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by olegrand1976** [documenso/documenso#1019] [General]
Hello,

  Have you planned translations into different languages of the interface? Particularly in French?

Thanks in advance.

Have a nice day.

**Comment by 1337-TIM** [comment]
hi there
I can make french and italian translations ;)

**Reply by allezxcendre** [reply]
Hi then you have succeeded to make a French version?

**Reply by 1337-TIM** [reply]
@allezxcendre  No .. I'm still waiting for "material" (means files and directories for languages) ... as far as I can see, nothing is ready for translations in this project (I must dmit I do not look very often at it)

**Comment by astrakid** [comment]
would like to participate for german translations. i encountered that german is not completely tranlsated.
